### PR TITLE
add single-instance compute-only starter

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: gtfisher <grant@chaosiq.io>

### CODE REVIEW

File: starlette-starter/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Python Starllete Web App Starter Kit
+
+Simple stateless web application using [Starlette][https://www.starlette.io/]
+
+To run locally
+
+* virtualenv ~/venv/starlet-starter
+* source ~/venv/starlet-starter/activate
+* pip install -r  requirements.txt
```
Comment (russmiles) [inline starlette-starter/README.md]: Additional space between -r and `requirements.txt`?

File: starlette-starter/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* pythin app.py
```
Comment (russmiles) [inline starlette-starter/README.md]: Typo

File: starlette-starter/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Python Starllete Web App Starter Kit
```
Comment (russmiles) [inline starlette-starter/README.md]: Is it worth specifying a version of Python required?

File: starlette-starter/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Click==7.0
```
Comment (russmiles) [inline starlette-starter/requirements.txt]: Is Click needed?

Comment (gtfisher) [reply starlette-starter/requirements.txt]: removed

**Pull request merged**